components/crossplane-control-plane/production/provider-config.yaml

```
@@ -0,0 +1,35 @@
+---
+apiVersion: kubernetes.crossplane.io/v1alpha1
+kind: ProviderConfig
+metadata:
+  name: eaas-prod-kubernetes-provider-config
```

See: https://github.com/konflux-ci/crossplane-control-plane/blob/429b2638313b298febcc24fb44b27a132f266722/README.md#L77
```suggestion
  name: eaas-kubernetes-provider-config
```